src/node-lib/curriculum-api-2023/queries/pupilUnitListing/pupilUnitListing.test.ts

```diff
@@ -0,0 +1,47 @@
+import {
+  programmeFieldsFixture,
+  syntheticUnitvariantsWithLessonIds,
+  unitDataFixture,
+  unitvariantFixture,
+} from "@oaknational/oak-curriculum-schema";
+
+import { pupilUnitListingQuery } from "./pupilUnitListing.query";
+
+import { PupilUnitListingQuery } from "@/node-lib/curriculum-api-2023/generated/sdk";
+import sdk from "@/node-lib/curriculum-api-2023/sdk";
+
+export const syntheticUnitvariantsWithLessonIdsFixture = ({
```

Comment:
I'm confused why this is here. Didn't you just do an update to oak-curriculum-schema to export this from there ?

Reply:
ill update now